packages/@okta/vuepress-site/docs/guides/add-an-external-idp/create-an-app-at-idp/oktatookta/appatidp.md

```diff
@@ -1,19 +1,29 @@
-When you are configuring federation between two Okta orgs using OpenID Connect:
+> **Note**: This guide will refer to the Okta org that represents the Identity Provider as **Org A** and the second Okta org as **Org B**.
 
-1. At the Okta org that represents the Identity Provider, select **Applications** and then click **Add Application**.
+1. At **Org A**, select **Applications** and then click **Add Application**.
 
-2. You need a trusted client, so select **Web** as the platform. OpenID Connect is the sign-in method by default.
+2. Click **Create New App**.
```

Comment:
All steps in the Developer Docs are written from the UI perspective of the Developer Console, not the Admin Console. We add a note if our users have to switch from the Developer Console to the Admin Console to perform a step

Thank you!

Reply:
Good to know!